crates/bitwarden-ssh/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+use rand::SeedableRng;
+use rand_chacha::ChaCha8Rng;
+use ssh_key::{Algorithm, HashAlg, LineEnding};
+
+pub mod error;
+pub mod models;
+
+pub enum KeyAlgorithm {
+    Ed25519,
+    Rsa3072,
+    Rsa4096,
+}
+
+pub fn generate_keypair(
```

Comment:
**suggestion:** `keypair` generally means an RSA keypair. In this case we generate an ssh keypair and we may benefit from including ssh in the name, to avoid confusion on the calling side where we can depend on bitwarden-crypto and bitwarden-ssh.